src/Analyzers/CSharp/Tests/CSharpAnalyzers.UnitTests.shproj

```diff
@@ -10,4 +10,11 @@
   <PropertyGroup />
   <Import Project="CSharpAnalyzers.UnitTests.projitems" Label="Shared" />
   <Import Project="$(MSBuildExtensionsPath32)\Microsoft\VisualStudio\v$(VisualStudioVersion)\CodeSharing\Microsoft.CodeSharing.CSharp.targets" />
+
+  <PropertyGroup>
+    <!-- We can use this to specify a path to a project that consumes this shared project
+         and our VS Code tasks will use that project when asked to build or run tests for
+         a file in this project. -->
+    <ProxyToConsumingProject>..\..\..\EditorFeatures\CSharpTest\Microsoft.CodeAnalysis.CSharp.EditorFeatures.UnitTests.csproj</ProxyToConsumingProject>
```

Comment:
I'm currently facing a unit test failure that happens only when test is run from CodeStyle, but not from EditorFeatures. Consider switching to CodeStyle for now?

Reply:
Ignore the comment. The bug causing failure in one layer but not the other could have happened the other way around.